services/TokenService.js

```diff
@@ -0,0 +1,11 @@
+const jwt = require('jsonwebtoken');
+
+function createToken(payload,expiresIn, callback){
+    jwt.sign(payload,process.env.JSON_SECRET, {expiresIn}, (err, token)=> {
+        callback(token);
```

Comment:
I believe we should stick to promises in place of callbacks, also in this use case, you should just return the token without a callback.